packages/cubejs-dremio-driver/driver/DremioQuery.js

```diff
@@ -59,11 +59,27 @@ class DremioQuery extends BaseQuery {
   }
 
   subtractInterval(date, interval) {
-    return `DATE_SUB(${date}, INTERVAL ${interval})`;
+    const intervalParts = interval.trim().split(' ');
```

Comment:
Unfortunately, intervals are a bit more complicated. You can get here interval values like:
* 1 hour 2 minutes
* 3 month
* 3 months 24 days 15 minutes
* 2 years 6 months
* Etc....

Please, have a look at some examples of interval processing in different dialects that might give you an idea how to process it here:
* https://github.com/cube-js/cube/blob/bb860165ec053b177dea79d174b9631b7b713b63/packages/cubejs-schema-compiler/src/adapter/MysqlQuery.ts#L62
* https://github.com/cube-js/cube/blob/master/packages/cubejs-schema-compiler/src/adapter/BigqueryQuery.ts#L180
* https://github.com/cube-js/cube/blob/master/packages/cubejs-schema-compiler/src/adapter/ClickHouseQuery.ts#L84

Have a look at `formatInterval()` implementations in these query dialects.
Hope this helps!